lxd/daemon_images.go

```diff
@@ -425,6 +429,11 @@ func (d *Daemon) ImageDownload(op *operation, server string, protocol string, ce
 			}
 		}
 
+		// Compatibility with older LXD servers
+		if info.Type == "" {
+			info.Type = "container"
```

Comment:
@stgraber same as above, using api.InstanceTypeContainer would make it easier to find in the future.